Tools/dollymix/zslice.py

```
@@ -0,0 +1,65 @@
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+"""
+metaentry = """    {{ "name": "{0}" }}"""
+metaterm = """
+  ]
+}"""
+
+states.append("icon")
+with  open(os.path.concat(path,"meta.json"), 'w') as metajson:
```

Pylint (3.3.7)

[error] Module 'posixpath' has no 'concat' member

(E1101)